docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: "3"
+networks:
+  db-data-net:
+    driver: "bridge"
+services:
+  pythonapp:
+    image: app:1.0
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "8080:8080"
+    networks:
+      - db-data-net
+    depends_on:
+      - mysql
+    restart: unless-stopped
+  mysql:
+    image: mysql:latest
+    build:
+      context: .
+      dockerfile: Dockerfile.mysql
```

Comment:
The `mysql` service specifies a build context and Dockerfile. Ensure that `Dockerfile.mysql` exists in the specified context directory. If you intend to use a pre-built image, you can remove the `build` section.